mac/makedistmac.sh

```diff
@@ -121,10 +121,7 @@ hdiutil convert dist/temp/rw.dmg -format UDZO -imagekey zlib-level=9 -o dist/$AP
 rm -f dist/temp/rw.dmg
 
 # add EULA
-hdiutil unflatten dist/$APPNAME.dmg
-DeRez -useDF $LIBRARYNAME/Main/Build/Mac/SLAResources.rsrc > dist/temp/sla.r
-Rez -a dist/temp/sla.r -o dist/$APPNAME.dmg
-hdiutil flatten dist/$APPNAME.dmg
+eulagise --license $LIBRARYNAME/Main/Build/license.txt --target dist/$APPNAME.dmg
```

Comment:
Yes, we need to have up to date documentation on how to set the builders for win/osx/linux